pyproject.toml

```diff
@@ -0,0 +1,59 @@
+[build-system]
+requires = ["setuptools>=68.0", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools]
+platforms = ["Linux x86, x86-64"]
+
+[tool.setuptools.packages.find]
+include = ["sopel_slap", "sopel_slap.*"]
+namespaces = false
+
+[project]
+name = "sopel-slap"
+version = "0.2.0.dev0"
+description = "Plugin for Sopel that lets users slap each other in fun ways"
+keywords = [
+  "sopel",
+  "plugin",
+  "slap",
+  "pranks",
+  "bot",
+  "irc",
+]
+maintainers = [
+  { name="dgw" },
+]
+authors = [
+  { name="dgw" },
+  { name="Michael Yanovich" },
+]
+
+readme = "README.md"
+license = { text="EFL-2.0" }
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Intended Audience :: Developers",
+    "Intended Audience :: System Administrators",
+    "License :: Eiffel Forum License (EFL)",
+    "License :: OSI Approved :: Eiffel Forum License",
+    "Operating System :: POSIX :: Linux",
+    "Programming Language :: Python :: 3.7",
```

Comment:
Oops, forgot to call this out. 3.7 is EOL so let's not bother advertising support.